README.md

```diff
@@ -2,102 +2,78 @@
 
-"Real" attacher talks to CSI over socket (`/run/csi/socket` by default, configurable by `-csi-address`). The attacher tries to connect for `-connection-timeout` (1 minute by default), allowing CSI driver to start and create its server socket a bit later.
-
-The attacher then:
-
-* Discovers the supported attacher name by `GetPluginInfo` calls. The attacher only processes `VolumeAttachment` instances that have `Attacher==GetPluginInfoResponse.Name`.
-* Uses `ControllerGetCapabilities` to find out if CSI driver supports `ControllerPublish` calls. It degrades to trivial mode if not.
-* Processes new/updated `VolumeAttachment` instances and attaches/detaches volumes:
-  * `VolumeAttachment` without `DeletionTimestamp`:
-    * Ignore `VolumeAttachment` that wants to attach PV with `DeletionTimestamp`.
-    * A finalizer is added to `VolumeAttachment` instance to preserve the object after deletion so we can detach the volume.
-    * A finalizer is added to referenced PV instance to preserve the PV. Attacher needs information from the PV to detach the volume.
-    * CSI `ControllerPublishVolume` is called.
-    * `AttachmentMetadata` is saved to `VolumeAttachment`.
-    * On any error, the `VolumeAttachment` is re-queued with exponential backoff.
-  * `VolumeAttachment` with `DeletionTimestamp`:
-    * CSI `ControllerUnpublishVolume` is called.
-    * A finalizer is removed from `VolumeAttachment`. At this point, the API server is going to delete this instance and "deleted `VolumeAttachment`" event will be received.
+## Overview
+The external-attacher is an external controller that monitors `VolumeAttachment` objects created by controller-manager and attaches/detaches volumes to/from nodes (i.e. calls `ControllerPublish`/`ControllerUnpublish`. Full design can be found at Kubernetes proposal at [container-storage-interface.md](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/container-storage-interface.md)
 
-* Processes deleted `VolumeAttachment` instances:
-  * Pokes PV queue with name of the detached PV. This triggers removal of finalizer on PV, if needed.
+## Compatibility
 
-* Processes added/updated PV to remove finalizer on PVs:
-  * Ignore PVs that don't have DeletionTimestamp.
-  * Checks that the PV is not used by any `VolumeAttachment` instance.
-  * Removes Attacher's finalizer on the PV if so.
-  * On any error, the PV is re-queued with exponential backoff.
+This information reflects the head of this branch.
 
-#### Concurrency
+| Compatible with CSI Version                                                                | Container Image                     | Min K8s Version |
+| ------------------------------------------------------------------------------------------ | ------------------------------------| --------------- |
+| [CSI Spec v0.3.0](https://github.com/container-storage-interface/spec/releases/tag/v0.3.0) | quay.io/k8scsi/csi-attacher:v0.2.0  | 1.10            |
```

Comment:
I think we can remove the first two lines. We only need to have the matrix for the current HEAD of the branch, while the csi-docs can point to different branches.

Reply:
Fixe